src/main/kotlin/org/jetbrains/bio/statistics/mixture/PoissonRegressionMixture.kt

```diff
@@ -0,0 +1,117 @@
+package org.jetbrains.bio.statistics.mixture
+import org.jetbrains.bio.dataframe.DataFrame
+import org.jetbrains.bio.experiments.fit.flipStatesIfNecessary
+import org.jetbrains.bio.statistics.Fitter
+import org.jetbrains.bio.statistics.Preprocessed
+import org.jetbrains.bio.statistics.emission.ConstantIntegerEmissionScheme
+import org.jetbrains.bio.statistics.emission.EmissionScheme
+import org.jetbrains.bio.statistics.emission.PoissonRegressionEmissionScheme
+import org.jetbrains.bio.viktor.F64Array
+import org.jetbrains.bio.viktor.asF64Array
+import kotlin.math.exp
+import kotlin.math.ln
+
+/**
+ * Mixture of 3 components:
+ * 0 - zero-inflated component
+ * 1 - LOW (poisson with small parameters)
+ * 2 - HIGH (poisson with high parameters)
+ *
+ * @author Elena Kartysheva
+ * @date 5/25/19
+ */
+class PoissonRegressionMixture(
```

Comment:
Can we have a test on sample/fit to ensure that model fits correctly? It looks like we have only integration tests on it.

Reply:
Sample-and-fit test added.